spacewalk/setup/spacewalk-setup.spec

```diff
@@ -278,6 +278,14 @@ if [ ! -f /etc/cobbler/settings.rpmsave -a -f /etc/cobbler/settings.before-migra
     touch /var/lib/cobbler/v2_migration_done
 fi
 
+# Wrong execution of v2 script happened, so we fix autoinstall attribute of collections.
+if test -f /etc/cobbler/v2_migration_done && ! grep -q autoinstall_fixed /etc/cobbler/v2_migration_done; then
```

Comment:
That's the wrong directory, the block above writes to `/var/lib/cobbler/v2_migration_done`

Reply:
My god :+1: thanks for catching it.

Reply:
Fixed.